api/iode.h

```diff
@@ -807,37 +804,38 @@ typedef char    ONAME[K_MAX_NAME + 1];
 typedef long    OSIZE;   /* JMP 23-05-00 */
 
 typedef struct  _kobj_ {
-    SWHDL       o_val;          // "position" in the memory -> to be passed to SW_getptr()
-    ONAME       o_name;         // name of the object
+    SWHDL       o_val;      /* SWHDL=long */ /* IODE64K */
+    ONAME       o_name;
     char        o_pad[3];
 } KOBJ;
 
 typedef struct _kdb_ {
-    KOBJ        *k_objs;                // map <position in the memory, object name>
```

Comment:
You also removed my comments here.

Additionnally, could replace my // ??? by the correct interpretation of the KDB member, please ?

Reply:
Same answer as above.

Reply:
Done